docs/development/contributing_guide.md

```diff
@@ -169,6 +169,21 @@ To increase the log level for the tests, set `SYNAPSE_TEST_LOG_LEVEL`:
 SYNAPSE_TEST_LOG_LEVEL=DEBUG trial tests
 ```
 
+By default, tests will use an in-memory SQLite database for test data. For additional
+help with debugging, one can use an on-disk SQLite database file instead, in order to
+review database state during and after running tests. This can be adjusted using the
```

Comment:
Can we explicitly mention that this database gets nuked at the start of an individual test? I.e. you only see the DB state for the current test, or the last test to finish.